npe2/cli.py

```diff
@@ -493,5 +499,28 @@ def cache(
                 typer.secho(f"{mf.name}: {mf.package_version}", fg=typer.colors.GREEN)
 
 
+@app.command()
+def compile(
+    src_dir: str,
+    output: Optional[Path] = typer.Option(
+        None,
+        "-o",
+        "--output",
+        exists=False,
+        help="If provided, will write manifest to filepath (must end with .yaml, "
+        ".json, or .toml). Otherwise, will print to stdout.",
+    ),
+    format: Format = typer.Option(
+        "yaml", "-f", "--format", help="Markdown format to use."
+    ),
+):
+    """Compile napari_plugin_engine plugins."""
```

Comment:
```suggestion
    """Compile @npe2.implements contributions to generate a manifest."""
```

Struggling a bit with the help text here.  Maybe this is ok?

Reply:
ohhhhh.  yep I'm so sorry. I missed that before!  thanks a lot for catching that!